### PR TITLE
Update mTLS tutorial to correct namespace for destination rules.

### PR DESCRIPTION
**Pull request by diemtvu**
Issue: istio/istio#12795

### CODE REVIEW

File: content/docs/tasks/security/authn-policy/index.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ EOF
 {{< /text >}}
 
 {{< tip >}}
+* Starting from 1.1, only destination rules in client namespace, server namespace and `global` namespace (default is `istio-system`) will be considered for a service, in that order.
```
Comment (geeknoid) [inline content/docs/tasks/security/authn-policy/index.md]: Starting with Istio 1.1, only

Comment (geeknoid) [reply content/docs/tasks/security/authn-policy/index.md]: in the client namespace

File: content/docs/tasks/security/authn-policy/index.md [hunk 2]
```diff
@@ -210,6 +212,10 @@ spec:
 EOF
 {{< /text >}}
 
+{{< tip >}}
+This destination rule is in the namespace of the server (`httpbin.legacy`), so it will be prefered over the global destination rule defined in `istio-system`
```
Comment (geeknoid) [inline content/docs/tasks/security/authn-policy/index.md]: preferred

**Comment by diemtvu** [comment]
Ping

**Pull request merged**